tools/azsdk-cli/Azure.Sdk.Tools.Cli/Services/Languages/JavaLanguageSpecificChecks.cs

```diff
@@ -38,21 +40,25 @@ public class JavaLanguageSpecificChecks : ILanguageSpecificChecks
 
     public JavaLanguageSpecificChecks(
         IProcessHelper processHelper,
-        ILogger<JavaLanguageSpecificChecks> logger)
+        IGitHelper gitHelper,
+        ILogger<JavaLanguageSpecificChecks> logger,
+        ICommonValidationHelpers commonValidationHelpers)
```

Comment:
.NET coding question, do usually helpers get used via dependency injection generally?

Reply:
This looks fine to me(But I am not a .NET expert). We are using DI for any of the helpers or services classes when it's instance registered in services unless all the methods in that helper is a static method. In this case, this common helper uses microagent instance, so this seems fine.